src/common/util/include/openvino/util/weights_path.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/util/common_util.hpp"
+
+namespace ov {
+namespace util {
+
+bool validate_weights_path(std::string& weights_path);
```

Comment:
`bool is_openvino_bin_file(std::string& weights_path);`
as it's not clear what it should return for ONNX files, which also contain weights

Reply:
Currently the solution is implemented only for the IR frontend. The logic should be changed once the ONNX frontend is enabled.